kasa/smartprotocol.py

```diff
@@ -321,86 +322,6 @@ async def close(self) -> None:
         await self._transport.close()
 
 
-class SnowflakeId:
```

Comment:
Is it intentional that the whole generator is now removed?

Reply:
Yes. It's not sent by the native apps and as far as I can tell from looking through the py100 lib it was introduced for some internal reason to do with the terminal ids but never utilised internally anyway. Considering the device flakiness when sending some unnecessary parameters it's best to drop it.